CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to the Shoreditch theme
+First of all, thank you for considering contributing to the Shoreditch theme!
+
+## Things to know before you start
+1. Bear in mind that the theme is currently in an **alpha stage** and at such many (even fundamental) things can be subject to change!
+2. Please read the [README](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/README.md) to get an overview of what the theme is and how it works
```

Comment:
I found some lines ends with "." but some others doesn't. please try to make it consistent